style.css

```diff
@@ -561,6 +580,26 @@ textarea {
     width: 70%;
   }
 
+  /* work section */
+  .container {
+    display: flex;
+    flex-direction: row;
+    margin: 30px 142px;
+  }
+
+  .desktop-img {
+    width: 100%;
+  }
+
+  .snap-shoot-one {
+    width: 100%;
+  }
+
+  .Project-one-header {
+    width: 100%;
+  }
+
+  /* About section  */
   .about-content {
     display: flex;
     flex-direction: row;
```

Comment:
- [ ] The content in the about section, should match the design in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848):

Right now your design looks like this:
![image](https://user-images.githubusercontent.com/20567503/178044174-f83521e4-542a-4bda-8cf6-73c6f3b95856.png)

And the one on Figma is like this:
![image](https://user-images.githubusercontent.com/20567503/178044225-f29a7ae3-f04c-410f-acae-009ccc1e7693.png)

Kindly fix it so they match 😁

Reply:
@mirouhml I will make it match now. Thank you🙏